package.json

```diff
@@ -22,6 +22,11 @@
     "react-dom": ">=16.0 <18.0",
     "replicache": ">=4.0.1 <10.0 || >7.0.0-beta <7.0.0 || >8.0.0-beta <8.0.0 || >9.0.0-beta <9.0.0"
```

Comment:
It could have always been an optional peer dependency (in older version of npm all peer dependencies were optional).  But we still need the peer dependency to specify what version of replicache this is compatible with.